samples/SQL/allowed-sql-database-collation/azurepolicy.json

```diff
@@ -0,0 +1,46 @@
+{
+   "properties": {
+     "displayName": "allowed-sql-database-collation",
+     "policyType": "Custom",
+     "mode": "All",
+     "metadata": {
+       "category": "SQL"      
+     },
+     "parameters": {
+       "listOfAllowedCollations": {
+         "type": "Array",
+         "metadata": {
+           "displayName": "Allowed Collation Names",
+           "description": "A list of approved collations"
+         }
+       }
+     },
+     "policyRule": {
+       "if": {
+         "allOf": [
+           {
+             "field": "type",
+             "equals": "Microsoft.SQL/servers/databases"
+           },
+           {
+             "field": "name",
+             "notEquals": "master"
+           },
+           {
+             "not": {
```

Comment:
The "collation" field is optional when creating a new SQL DB and has some default.  The policy will need to take that into account and not deny if the default is in the listOfAllowedCollations.  I am not aware of what the default is but I'm sure its either in SQLs docs or can be tested by creating a new DB

Reply:
Hey, the default collation is SQL_Latin1_General_CP1_CI_AS. If this collation is not whitelisted as part of the listOfAllowedCollations, then the database creation will fail (denied by policy). The idea I had about this policy was to precisely put some governance around the exact desired collation/s , and don't allow defaults unless explicitly indicated (unexpected sort operations will happen), sounds good?